backend/test/card_game/api_play_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns card-game.api-play-test
+  (:require [expectations.clojure.test :refer :all]
+            [card-game.api.base :as api]
+            [configs :as configs]))
+
+(defexpect playing-cards
+  (expect
+    #(empty? (filter (fn [e] (nil? (:power e))) %))
+    (-> (api/create-game)
+        :hand))
+
+  ; Cards are removed from hands upon being played
+  (expect
+    #(= (count (:hand %)) (dec (count (configs/ini-hand))))
+    (let [game (api/create-game)
+          opponent (api/add-player (:game-id game))]
+      (api/play-card-as-player (:game-id game) (:player-id opponent) 0 0)
+      (api/play-card-as-player (:game-id game) (:player-id game) 0 0)))
+
+  ; Card is not removed if opponent has not yet played
+  (expect
+    #(= (count (:hand %)) (count (configs/ini-hand)))
+    (let [game (api/create-game)
+          opponent (api/add-player (:game-id game))]
+      (do
+        (api/play-card-as-player (:game-id game) (:player-id game) 0 0)
+        (api/get-game (:game-id game) (:player-id game)))))
+
+  ; Fetching the game as a player returns one less card after a play
+  (expect
+    #(= (count (:hand %)) (dec (count (configs/ini-hand))))
+    (let [game (api/create-game)
+          opponent (api/add-player (:game-id game))]
+      (do
+        (api/play-card-as-player (:game-id game) (:player-id opponent) 0 0)
+        (api/play-card-as-player (:game-id game) (:player-id game) 0 0)
+        (api/get-game (:game-id game) (:player-id game)))))
+
+  ; card played is owned by self
```

Comment:
This test and the next one can be merged into one "Card is owned by the player that played it"

Two approaches to do it:

1. Extract the `(do` bodies into a `(let` with both the expects *inside* the `(let` (there is no reason `(expect` needs to be a direct child of `(defexpect` I believe.
2. Make one lone expect with both acceptance functions inside an `(and` (but we would then not now which of the two cases failed upon a failure)

Reply:
I understand that's because you don't like to testes that are *the same game* with different expects. Still, I don't think it will then be more readable doing that.
Anyway, I'll add more stuff to the `let` of all those tests.

Reply:
This one is not about readability, it's about duplication. If we change, for example, how games are generated, we would need to duplicate the change here unnecessarily, since we really use the same game.